### PR TITLE
fix for odataType in educationClass-Post-Assignments

### PR DESCRIPTION
**Pull request by sgoarla**
The odata type value is wrong in  educationClass-Post-Assignments for both graph beta and v1.0. this change is to fix that value.
This is the work item(146426) https://1edu.visualstudio.com/Homeroom/_workitems/edit/146426 

the current version of this doc is : https://docs.microsoft.com/en-us/graph/api/educationclass-post-assignments?view=graph-rest-beta&tabs=http

### CODE REVIEW

**Comment by isvargasmsft** [comment]
Hello @sgoarla, 

What is the status of this PR? When you are ready for a content review, please add the label "ready to merge". 

For details on the content review process, please visit: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211773/Content-workflow?anchor=content-review-process-summary

File: api-reference/beta/api/educationclass-post-assignments.md [hunk 1]
```diff
@@ -65,11 +65,11 @@ Content-type: application/json
         "content": "Read chapter 4"
     },
     "grading": {
-        "@odata.type": "#microsoft.graph.educationAssignmentPointsGradeType",
+        "@odata.type": "#microsoft.education.assignments.api.educationAssignmentPointsGradeType",
```
Comment (Lauragra) [inline api-reference/beta/api/educationclass-post-assignments.md:68]: The OData type associated with this property is: microsoft.graph.educationAssignmentGradeType.

File: api-reference/v1.0/api/educationclass-post-assignment.md [hunk 2]
```diff
@@ -62,11 +62,11 @@ Content-type: application/json
         "content": "Read chapter 4"
     },
     "grading": {
-        "@odata.type": "#microsoft.graph.educationAssignmentPointsGradeType",
+        "@odata.type": "#microsoft.education.assignments.api.educationAssignmentPointsGradeType",
```
Comment (Lauragra) [inline api-reference/v1.0/api/educationclass-post-assignment.md:65]: The OData type associated with this property is: microsoft.graph.educationAssignmentGradeType.

File: api-reference/v1.0/api/educationclass-post-assignment.md [hunk 3]
```diff
@@ -62,11 +62,11 @@ Content-type: application/json
...
         "maxPoints": 50
     },
     "assignTo": {
-        "@odata.type": "#microsoft.graph.educationAssignmentClassRecipient"
+        "@odata.type": "#microsoft.education.assignments.api.educationAssignmentPointsGradeType"
```
Comment (Lauragra) [inline api-reference/v1.0/api/educationclass-post-assignment.md:69]: The OData type associated with this property is: microsoft.graph.educationAssignmentRecipient.

File: api-reference/beta/api/educationclass-post-assignments.md (hunk 1 above)
Comment (sgoarla) [reply api-reference/beta/api/educationclass-post-assignments.md:68]: the source from which I got says it should be: microsoft.education.assignments.api.educationAssignmentPointsGradeType, so you mean to keep it as : microsoft.graph.educationAssignmentGradeType , right ?? sure, I will reconfirm it and update it.. Thanks!

File: api-reference/v1.0/api/educationclass-post-assignment.md (hunk 2 above)
Comment (sgoarla) [reply api-reference/v1.0/api/educationclass-post-assignment.md:65]: same comment as above..

File: api-reference/beta/api/educationclass-post-assignments.md (hunk 1 above)
Comment (Lauragra) [reply api-reference/beta/api/educationclass-post-assignments.md:68]: Yes, I referred to the metadata and this is the listed type.

Comment (sgoarla) [reply api-reference/beta/api/educationclass-post-assignments.md]: updated to the right one..

**Review by Lauragra — APPROVED** [review]

**Pull request merged**